account_credit_control/views/credit_control_line.xml

```diff
@@ -118,6 +111,12 @@
                     <filter domain='[]'
                             context="{'group_by': 'manually_overridden'}"
                             icon="terp-document-new" string="Manual change"/>
+                    <separator orientation="vertical"/>
+                    <field name="partner_id"/>
+                    <field name="account_id"/>
+                    <field name="invoice_id"/>
+                    <field name="level"/>
+                    <field name="policy_level_id"/>
```

Comment:
The fields do appear when searching for a text, so I now grouped them before filters.